turbinia/processors/mount_local.py

```diff
@@ -30,6 +30,48 @@
 RETRY_MAX = 10
 
 
+def GetDiskSize(source_path):
+  """Gets the size of disk evidence in bytes.
+
+  Tries using blockdev to query the size of block devices, and falls back on
```

Comment:
dfVFS can get partition sizes by reading from the volume system, but I don't believe it can get the size of the whole disk (at least I don't think it exposes that).